editor/src/components/canvas/controls/select-mode/select-mode-hooks.tsx

```diff
@@ -41,22 +41,26 @@ import { useWindowToCanvasCoordinates } from '../../dom-lookup-hooks'
 import { useInsertModeSelectAndHover } from './insert-mode-hooks'
 import { WindowMousePositionRaw } from '../../../../utils/global-positions'
 import { isFeatureEnabled } from '../../../../utils/feature-switches'
+import { createInteractionViaMouse } from '../../canvas-strategies/interaction-state'
+import { Modifier } from '../../../../utils/modifiers'
 
 const DRAG_START_TRESHOLD = 2
 
 export function isResizing(editorState: EditorState): boolean {
   const dragState = editorState.canvas.dragState
   return (
-    dragState?.type === 'RESIZE_DRAG_STATE' &&
-    getDragStateDrag(dragState, editorState.canvas.resizeOptions) != null
+    (dragState?.type === 'RESIZE_DRAG_STATE' &&
+      getDragStateDrag(dragState, editorState.canvas.resizeOptions) != null) ||
+    editorState.canvas.interactionSession != null
   )
 }
 
 export function isDragging(editorState: EditorState): boolean {
   const dragState = editorState.canvas.dragState
   return (
-    dragState?.type === 'MOVE_DRAG_STATE' &&
-    getDragStateDrag(dragState, editorState.canvas.resizeOptions) != null
+    (dragState?.type === 'MOVE_DRAG_STATE' &&
+      getDragStateDrag(dragState, editorState.canvas.resizeOptions) != null) ||
+    editorState.canvas.interactionSession != null
```

Comment:
This doesn't _feel_ right as not every interaction is dragging, maybe needs a `TODO` to indicate it needs removing/replacing later?

Reply:
added a comment in 6b90b5b8